app/lib/hesa/parsers/itt_record.rb

```diff
@@ -15,13 +15,13 @@ class IttRecord
         course_subject_one: "F_SBJCA1",
         course_subject_two: "F_SBJCA2",
         course_subject_three: "F_SBJCA3",
-        itt_start_date: "F_COMDATE",
+        commencement_date: "F_COMDATE",
```

Comment:
Would it be worth naming this `hesa_comdate` or `hesa_com_date` to indicate it's a hesa thing - and then later we have some logic to pick between this and the other date.

Reply:
as above.